src/MIDebugEngine/AD7.Impl/AD7StackFrame.cs

```diff
@@ -399,7 +398,7 @@ int IDebugStackFrame2.EnumProperties(enum_DEBUGPROP_INFO_FLAGS dwFields, uint nR
                     guidFilter == AD7Guids.guidFilterLocals ||
                     guidFilter == AD7Guids.guidFilterLocalsPlusArgs)
                 {
```

Comment:
Delete this whole if block -- you don't need to pre-call `GetLocalsAndParameters` any more since each of the `CreateXyzProperties` does this.